blivet/actionlist.py

```diff
@@ -259,10 +259,9 @@ def _pre_process(self, devices=None):
         for action in self._actions:
             log.debug("action: %s", action)
 
-            # Remove lvm filters for devices we are operating on
-            lvm.lvm_cc_removeFilterRejectRegexp(action.device.name)
             for device in (d for d in devices if d.depends_on(action.device)):
-                lvm.lvm_cc_removeFilterRejectRegexp(device.name)
```

Comment:
It will be wonderful to see this code go away. It made so much noise in the logs, but I never took the time to quiet it down.